style.css

```diff
@@ -0,0 +1,220 @@
+*{
+    box-sizing: border-box;
+}
+
+body{
+    margin: 0;
+    font-size: 16px;
+    font-family: Helvetica, sans-serif;
+}
+
+.topContainer{
+    position: relative;
+    margin: 0 auto;
+    margin-top: 100px;
+    height: 650px;
+    width: 85%;
+    overflow: auto;
+    text-align: center;
+}
+
+.container{
+    position: relative;
+    margin: 0 auto;
+    padding-top: 50px;
+    height: 650px;
+    width: 85%;
+    overflow: auto;
+    text-align: center;
+}
+
+.breatherContainer{
+    position: relative;
+    margin: 0 auto;
+    height: 350px;
+    width: 100%;
+    overflow: auto;
+    text-align: center;
+}
+
+.bottomContainer{
+    position: relative;
+    margin: 0 auto;
+    height: 250px;
+    width: 85%;
+    overflow: auto;
+    text-align: center;
+}
+
```

Comment:
Lots of the classes above have a lot of common traits repeated, it might be more efficient to add those in one class.